README.md

```diff
@@ -1,4 +1,6 @@
 easy-datacite
 =============
 
+[![Build Status](https://travis-ci.org/DANS-KNAW/easy-datacite.svg?branch=master)](https://travis-ci.org/DANS-KNAW/easy-datacite)
```

Comment:
Agree with both. Feel free to do a pull request on the archetype project! Regarding non-public: every project starts off this way during initial development. Then, at some random point in time we make it public...